### DISCUSSION TITLE
Is there a way to create a custom handler for 500 responses?

### DISCUSSION THREAD

**Discussion by levu42** [livewire/livewire#5038] [1. Help]
Whenever there's a 500 response in a Livewire XHR request, I get the overlay with a 500 - is there a way to create a custom handler for that in JS to use my own display logic that fits to the look & feel of the application?

**Comment by x7ryan** [comment]
You can customize the 500 error page [in Laravel](https://laravel.com/docs/9.x/errors#custom-http-error-pages). It still shows as an overlay. Idk that you can get it show just a regular full screen error page that's not overlaid. Perhaps you could get Laravel to make its 500 error page a redirect to a full page error page?

**Reply by levu42** [reply]
I don't even want a full screen error page, I want a small toast in the corner saying "ooos something went wrong". So I'd like to tell Livewire "instead of you rendering the error please let me take care of it with this error handler I have here"

**Comment by joshhanley** [comment]
@levu42 not that I know of. I know there is now a javascript hook for handling the session timeout, but I don't think there is one for 500 errors.
